libs/accounts/recovery-phone/src/lib/recovery-phone.service.spec.ts

```diff
@@ -94,4 +100,20 @@ describe('RecoveryPhoneService', () => {
       mockError
     );
   });
+
+  describe('removePhoneNumber', () => {
+    it('should remove a phone number', async () => {
+      mockRecoveryPhoneManager.removePhoneNumber.mockResolvedValueOnce(true);
+      const result = await service.removePhoneNumber(uid);
+      expect(result).toBeTruthy();
+      expect(mockRecoveryPhoneManager.removePhoneNumber).toBeCalledWith(uid);
+    });
+
+    it('should throw if phone number not found', () => {
+      mockRecoveryPhoneManager.removePhoneNumber.mockRejectedValueOnce(
+        new RecoveryNumberNotExistsError(uid)
+      );
+      expect(service.removePhoneNumber(uid)).rejects.toThrow();
```

Comment:
tiny nit, check that it throws a RecoveryNumberNotExistsError and not a different error.